### PR TITLE
stormlib 9.21

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

**Comment by mistydemeo** [comment]
Patch doesn't apply.

**Comment by mistydemeo** [comment]
Looks like we're not getting a dylib in this update, just a `.a`.

File: stormlib.rb [hunk 1]
```diff
@@ -19,7 +18,9 @@ class Stormlib < Formula
   patch :DATA
 
   def install
-    system "cmake", "CMakeLists.txt", "-DWITH_STATIC=YES", *std_cmake_args
+    system "cmake", "..", *std_cmake_args
```
Comment (mistydemeo) [inline stormlib.rb]: Probably don't want to pass `".."` here?

**Comment by mistydemeo** [comment]
Thanks!

**Pull request closed without merging**